lib/cocoapods/version_metadata.rb

```diff
@@ -1,11 +1,13 @@
 module Pod
   module VersionMetadata
+    CACHE_VERSION = '001'.freeze
+
     def self.gem_version
       Pod::VERSION
     end
 
     def self.project_cache_version
-      VersionMetadata.gem_version
+      "#{VersionMetadata.gem_version}.project-cache.#{CACHE_VERSION}"
```

Comment:
A version bump here isn't completely necessary for incremental, but they wouldn't get the updated `Pods.xcodeproj` structure until aggregate targets were dirtied. Might as well just force the clean install for those who want this.